### ISSUE TITLE
Comboboxitem tooltip bug

### ISSUE THREAD

**Issue by AntonNah** [dotnet/wpf#5716]
* .NET Core Version: NET 6, NET 5
* Windows version: windiws 10 1803

NET 6
![1](https://user-images.githubusercontent.com/33244866/142616581-e170afcf-d2f6-4349-9416-a4c6519e3d64.gif)
 
NET 5
![3](https://user-images.githubusercontent.com/33244866/142617362-70ce3eb4-f1f0-44e6-b047-7e5c3da1f4f8.gif)

   ```
<Grid>
        <ComboBox Width="100" Height="20">
            <ComboBoxItem ToolTip="1">13.42</ComboBoxItem>
            <ComboBoxItem ToolTip="2">15.82</ComboBoxItem>
        </ComboBox>
    </Grid>
```

**Comment by lindexi** [comment]
See #4612

**Comment by singhashish-wpf** [comment]
@AntonNah Could you please specify the exact .Net Core version you are using?

**Comment by Yoooi0** [comment]
@singhashish-wpf 
I think he got the naming confused, the issue is only happening in .NET6, in .NET5 the behavior is correct.

I have the exact same issue but when using Popup's. I think this is related since ComboBox also uses a Popup.
The issue seems to be coming from calling `Mouse.Capture(control, CaptureMode.SubTree);` on the parent control that owns the popup.

Sample control code:
```xaml
<UserControl x:Class="WpfIssueControl"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             MouseLeftButtonDown="UserControl_MouseLeftButtonDown">
    <Grid>
        <ToggleButton x:Name="Button" 
                      Checked="Button_Checked" 
                      Unchecked="Button_Unchecked"/>

        <Popup x:Name="Popup" 
               IsOpen="{Binding IsChecked, ElementName=Button}">
            <Border Background="Gray">
                <TextBlock Text="TextBlock" 
                           ToolTip="ToolTip"
                           ToolTipService.InitialShowDelay="0"/>
            </Border>
        </Popup>
    </Grid>
</UserControl>
```

```csharp
public partial class WpfIssueControl : UserControl
{
    public WpfIssueControl() => InitializeComponent();
    private void Button_Checked(object sender, RoutedEventArgs e) => Mouse.Capture(this, CaptureMode.SubTree);
    private void Button_Unchecked(object sender, RoutedEventArgs e) => Mouse.Capture(null);
    private void UserControl_MouseLeftButtonDown(object sender, MouseButtonEventArgs e) => Button.IsChecked = false;
}
```

Removing the `Mouse.Capture(this, CaptureMode.SubTree);` call also fixes the issue and makes the tooltip behave normally, obviously that is not a proper fix.
As in OP the issue only happens when targeting `net6.0-windows`, switching to `net5.0-windows` also makes the tooltip behave normally.

**Comment by AntonNah** [comment]
Net5 also not correct as seen in the gif.

**Comment by Yoooi0** [comment]
> Net5 also not correct as seen in the gif.

If you mean that there is a delay I think that should be a different issue.
You didnt explain anything in the OP so I thought this was about .NET6 tooltip not working at all.

**Comment by amirburbea** [comment]
I made a repro of this bug: https://github.com/amirburbea/PopupRepro

Run this app under .NET 5 and .NET 6 and you'll see that ListBoxItem has no issues with tooltip, but ComboBoxItem does.

**Comment by cheesi** [comment]
I can confirm this issue with .NET 6.0.101.

**Comment by cheesi** [comment]
As this is closed now: will this fix be delivered with .NET 6.0.2?

**Comment by singhashish-wpf** [comment]
@cheesi It is most likely that this will be delivered with the next servicing release for 6.0; This still depends on some other factors, Will keep this posted once there is a confirmation.

**Comment by Mgamerz** [comment]
Should this be re-opened, since the issue seems to be worse now than it was before, and it's been a servicing release (6.0.3 came out in march)?

**Comment by Dunge** [comment]
Having the same issue with 6.0.3. Works on my home (Win11) and work (Win10) computer fine, but customer (Win10) recorded a video showing the same behavior on his. Not sure what's the difference between machine to cause the issue, but it was working in NET5 and started showing up when upgrading to NET6.

**Comment by Mgamerz** [comment]
From others comments (and a quick test of mine) it seems to have to do with
dpi scaling.

**Comment by lindexi** [comment]
@Mgamerz  Do you means https://github.com/dotnet/wpf/issues/6289 ?

**Comment by Mgamerz** [comment]
Yes, as well as reports here.

**Comment by toristorii** [comment]
.NetCore 6.0.3 here and having the same problem with tooltips on pretty much anything, even a button.  

Steps I did to repro:
Created a new .net core 6 wpf app, in Visual Studio.  
Added a button to the grid in MainWindow.xaml as follows:
   <Button Content="Test" Height="25" Width="100" ToolTip="Hello World"/>
Run.  
Mouse over the button and 95% of the time it only appears for a fraction of a second.

Change to .netcore 5 and it works correctly.
